### DISCUSSION TITLE
Using workspace wildcards with turbo-ignore?

### DISCUSSION THREAD

**Discussion by AlexKMarshall** [vercel/turborepo#5643] [Help] [answered]
With the main `turbo` command we can use wildcards to filter the workspaces that are run e.g. `turbo run test --filter="./packages/*"` to run the test command for everything in the packages folder.

With `turbo-ignore` the docs only mention using package names directly https://turbo.build/repo/docs/core-concepts/monorepos/skipping-tasks. E.g. `npx turbo-ignore web --task test` to check whether the test task in the web workspace would run.

Is there a way to pass wildcards to turbo-ignore too? I have a whole series of small utility packages that I test with `turbo run test --filter="./packages/*"` and if none have changed I'd like to skip that run.

Running `npx turbo-ignore "./packages/*" --task test" looks as though it only checks the first package that meets that wildcard, rather than including all of them

**Comment by anthonyshew** [comment, marked as answer]
This is/was probably a bug. If you're still seeing this, please open an issue so we can get it fixed. 🙏
